storage/clickhousespanstore/writer.go

```diff
@@ -82,10 +97,14 @@ func (w *SpanWriter) registerMetrics() {
 
 func (w *SpanWriter) backgroundWriter() {
 	batch := make([]*model.Span, 0, w.size)
+	maxSize := 4 * w.size
```

Comment:
why `4*`?

Reply:
it seems like a magic number :)

Reply:
I don't know what to use here. Maybe original or twice the original size will do.